.github/workflows/zowe-cli.yml

```diff
@@ -121,7 +128,6 @@ jobs:
       run: cargo test ${{ github.event.inputs.binary-type == 'release' && '--release' || '' }} --manifest-path=zowex/Cargo.toml -- --nocapture
 
     - name: Unit Tests
-      id: unit
```

Comment:
I'm guessing there's no need to have IDs if we don't ask for their outcome 😋

Reply:
That was my thought as well 🙂 Doing a bit of cleanup on the workflow like we did for the Imperative one.